vyper/context/types/utils.py

```diff
@@ -91,7 +91,8 @@ def get_type_from_abi(
     type_string = abi_type["type"]
     if type_string == "fixed168x10":
         type_string = "decimal"
-    # TODO string and bytes
```

Comment:
Why was this note there?

Reply:
in anticipation of this, I guess :smile: